packages/host/app/lib/stack-item.ts

```diff
@@ -99,14 +99,17 @@ export class StackItem {
     if (!this.cardResource || !this.cardResource.card) {
       return;
     }
-    let cardDef = this.cardResource.card.constructor;
+    let cardDef = this.cardResource.card;
```

Comment:
This is now a card instance and not a CardDef, correct? If so, the local variable should be renamed.